canopy/design-system/src/App.js

```diff
@@ -62,14 +61,16 @@ const tabs = [
   }
 ];
 
-const Introduction = lazy(() => importMDX('./views/Introduction.mdx'));
+const Introduction = lazy(() =>
```

Comment:
What is the difference between this commit and "Add introduction page"?

Reply:
This one uses the custom babel loader instead of the loader provided by mdx.macro because that one didn't work exactly right

Reply:
Can you squash those changes together? Since like 65 was added in a different commit of this PR